docs/general/ad-filtering/create-own-filters.md

```diff
@@ -1177,6 +1198,7 @@ In that case, the `$badfilter` rule will disable the corresponding rule for doma
 :::info Compatibility
 
 Rules with `$badfilter` modifier are not supported by AdGuard Content Blocker.
+In the current implementation of AdGuard Browser Extension MV3, it works across all filters, but it does not cover rules with `$domain` instersections.
```

Comment:
@105th what is `$domain` intersections

Reply:
@SeverAnna we should place here some examples to illustrate that we applied $badfilter in DNR way only if it "full cancel" source rule. if it just a partial cancelling, we cannot calculate it. examples - https://github.com/AdguardTeam/tsurlfilter/tree/epic/tswebextension/packages/tsurlfilter/src/rules/declarative-converter#badfilter